recipes/pelicanplatform/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -ex
+
+export CGO_ENABLED=0
+
+go generate ./...
+
+cd "origin_ui/src"
+npm install
+npm run build
+
+cd "${SRC_DIR}"
+go build \
+  -a \
+  -ldflags "-w -s -X main.version=${PKG_VERSION} -X main.commit=v${PKG_VERSION} -X main.date=$(date -u +"%Y-%m-%dT%H:%M:%SZ") -X main.builtBy=conda-forge" \
+  -tags forceposix \
+  -p ${CPU_COUNT} \
+  -v \
+  -o "${PREFIX}/bin/pelican" \
+  ./cmd
+
+# generate the license pack (which fails because of an unknown license)
+go get ./...
+go-licenses save ./... --save_path license-files || true
```

Comment:
Manually look up the failed license and ignore this package in this command.
```suggestion
go-licenses save ./... --save_path license-files
```

If you cd into the `cmd` directory, you probably can limit the scope of licenses scanned (it may exclude the test licenses then).